src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -3,9 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Diagnostics;
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
+using System.Reflection;
```

Comment:
Is it possible to change the anonymous type so that it doesn't inherit the __BlockReflectionAttribute?